tools/audit/repo-mapper/Dockerfile

```diff
@@ -6,14 +6,28 @@ RUN         pip3 --version
 RUN         ansible --version
 # RUN         pip3 install --user --ignore-installed ansible==2.9.6
 
+# test out oc command:
+# specify the version string of the oc release
+ENV         OC_VERSION v3.11.0
+ENV         OC_RELEASE openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit
 USER        0
 WORKDIR     /opt/
 COPY        ansible/  .
+# install the oc client tools
+RUN         curl -L --silent -o oc.tar.gz https://github.com/openshift/origin/releases/download/v3.11.0/openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz && \
+            tar -xzf oc.tar.gz --strip 1 && \
+            cp oc /usr/bin/ && \
+            chmod +x oc
+
+RUN         cp oc /usr/bin/
 
 # Set permissions for OpenShift on webhook artifacts
 RUN         chgrp -R 0 /opt && \
             chmod -R g+rwX /opt && \
             chmod 664 /etc/passwd
 
+# Port for Caddy:
+EXPOSE      9000
```

Comment:
don't think this is needed since we'd run a multi container pod?

Reply:
Yea I can also remove the service and route template if you've got it included in web